src/tools/namespace_tool.cpp

```diff
@@ -94,15 +96,22 @@ int NameSpaceTool::RunCommand(const std::string &cmd) {
             return 0;
         }
     } else if (cmd == kCleanRecycleCmd) {
+        uint64_t expireTime;
+        if (!::curve::common::StringToTime(FLAGS_expireTime, &expireTime)) {
+            std::cout << "Clean recyclebin: invalid expireTime!" << std::endl;
+            return 0;
+        }
+
+        auto dirname = fileName.empty() ? "/" : fileName;
         if (FLAGS_isTest) {
-            return core_->CleanRecycleBin(fileName);
+            return core_->CleanRecycleBin(dirname, expireTime);
         }
         std::cout << "Are you sure you want to clean the RecycleBin?"
```

Comment:
std::cout << "Are you sure you want to clean the RecycleBin?"
                  << "(yes/no)" << std::endl;

the tip shuold give the dirname

Reply:
> std::cout << "Are you sure you want to clean the RecycleBin?"
> << "(yes/no)" << std::endl;
> 
> the tip shuold give the dirname

done